app/code/Magento/Cms/Model/ResourceModel/Block.php

```diff
@@ -91,6 +92,10 @@ protected function _beforeSave(AbstractModel $object)
                 __('A block identifier with the same properties already exists in the selected store.')
             );
         }
+
+        $date = (new \DateTime())->format(DateTime::DATETIME_PHP_FORMAT);
```

Comment:
Please, use framework implementation for date value \Magento\Framework\Stdlib\DateTime\DateTime::date